src/pages/PersonPage/PersonPage.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+export const PersonPage: React.FC<{ person: Person }> = ({
+  person: {
+    name,
+    born,
+    died,
+    fatherName,
+    motherName,
+    sex,
+    slug,
+    mother,
+    father,
+  },
+}) => {
+  const { personSlug } = useParams();
+
+  const renderParrents = (
+    parrentPerson: Person | null = null,
+    parrentName: string | null,
+  ) => {
+    if (parrentPerson) {
+      return (
+        <Link
+          className={parrentPerson.sex === 'f' ? 'has-text-danger' : ''}
+          to={parrentPerson.slug}
+        >
+          {parrentName}
+        </Link>
+      );
+    }
+
+    return parrentName || '-';
+  };
```

Comment:
Looks like seperate component instead render function

Reply:
not fixed